alembic/versions/54f755dbdb6f_add_category_and_time_lost_type_.py

```diff
@@ -0,0 +1,55 @@
+"""add category and time_lost_type parameters
```

Comment:
Please start all comments with a Capital letter and end in a `.`.

Reply:
If you look at other migrations on https://github.com/lsst-sqre/narrativelog/tree/main/alembic/versions you will see none use capital letter neither `.`. I think this is because that comment is being added with an alembic command, you can check more info [here](https://alembic.sqlalchemy.org/en/latest/tutorial.html#create-a-migration-script).